src/main/java/com/jagrosh/vortex/commands/automod/WhitelistInvitesCmd.java

```diff
@@ -0,0 +1,116 @@
+
+    public WhitelistInvitesCmd(Vortex vortex)
+    {
+        this.vortex = vortex;
+        this.name = "whitelist";
+        this.guildOnly = true;
+        this.category = new Category("AutoMod");
+        this.arguments = "<ADD GUILD_ID[ GUILD_ID...]|REMOVE GUILD_ID[ GUILD_ID...]|SHOW>";
+        this.help = "if strikes for invites are enabled, add/remove whitelisted guilds";
+        this.userPermissions = new Permission[]{Permission.MANAGE_SERVER};
+    }
+
+    @Override
+    protected void execute(CommandEvent event)
+    {
+        String[] args = event.getArgs().toLowerCase().split("\\s+");
+        if(event.getArgs().equalsIgnoreCase("show") || (args.length > 1 && (args[0].equals("add") || args[0].equals("remove"))))
+        {
+            if(event.getArgs().equalsIgnoreCase("show"))
+            {
+                List<Long> currentWL = vortex.getDatabase().inviteWhitelist.readWhitelist(event.getGuild());
+                event.replySuccess("Whitelisted Guild IDs:\n" + (currentWL.isEmpty() ? "None" :
+                        currentWL.stream().map(String::valueOf).collect(Collectors.joining(", "))));
+            }
+            else
+            {
+                List<Long> guildIds = readIds(args);
+                if(guildIds == null)
+                {
+                    event.replyWarning("Invalid Guild-ID(s) provided!");
+                    return;
+                }
+                if(args[0].equals("add"))
+                {
+                    if(guildIds.size() > 1)
+                    {
+                        vortex.getDatabase().inviteWhitelist.addAllToWhitelist(event.getGuild(), guildIds);
+                    }
+                    else if(!vortex.getDatabase().inviteWhitelist.addToWhitelist(event.getGuild(), guildIds.get(0)))
+                    {
+                        event.replyWarning("Given Guild was already whitelisted");
+                        return;
+                    }
+                }
+                else
+                {
+                    if(guildIds.size() > 1)
+                    {
+                        vortex.getDatabase().inviteWhitelist.removeAllFromWhitelist(event.getGuild(), guildIds);
+                    }
+                    else if(!vortex.getDatabase().inviteWhitelist.removeFromWhitelist(event.getGuild(), guildIds.get(0)))
+                    {
+                        event.replyWarning("Given Guild was not whitelisted");
+                        return;
+                    }
+                }
+                event.replySuccess("Whitelist has been modified");
+            }
+        }
+        else
+        {
+            event.replyWarning(DESCRIPTION+"\nValid options are `ADD GUILD_ID[ GUILD_ID...]`, `REMOVE GUILD_ID[ GUILD_ID...]` and `SHOW`");
+        }
+        
+    }
+
+    private List<Long> readIds(String[] args)
```

Comment:
I also want there to be a check here to only allow longs greater than a certain amount (maybe `10000000000000000`)